index.js

```diff
@@ -0,0 +1,123 @@
+// Import stylesheets
+import './style.scss';
+
+var redDisplay = document.querySelector(".red");
+var greenDisplay = document.querySelector(".green");
+var blueDisplay = document.querySelector(".blue");
+var easyButton = document.querySelector(".easy");
+var hardButton = document.querySelector(".hard");
+var newColors = document.querySelector(".new-game");
+var squaresList = document.querySelectorAll("li.square");
+var modeButtons = document.querySelectorAll(".mode");
+
+var BG_COLOR = "#4e82b4";
+var isEasyGame = false;
+var isGameOver = false;
+
+function startGame() {
+  isGameOver = false;
+  document.querySelector("header").style.backgroundColor = BG_COLOR;
```

Comment:
Lets declare it up there with all the rest (you also use it twice so it will be better to not query the DOM twice).